src/cms/components/parks/page-header.json

```diff
@@ -0,0 +1,26 @@
+{
+  "collectionName": "components_parks_page_headers",
+  "info": {
+    "name": "PageHeader",
```

Comment:
Forgot to say we can switch the landing page over to this as well, going to be super helpful 🕺